docs/bigquery/snippets.py

```diff
@@ -1365,6 +1365,126 @@ def test_load_table_from_uri_truncate(client, to_delete):
+        bigquery.SchemaField('full_name', 'STRING', mode='REQUIRED'),
+    ]
+    table = client.create_table(bigquery.Table(table_ref, schema=old_schema))
+
+    # Configure the load job to append the data to a destination table,
+    # allowing field addition
+    job_config = bigquery.LoadJobConfig()
+    job_config.write_disposition = bigquery.WriteDisposition.WRITE_APPEND
+    job_config.schema_update_options = [
+        bigquery.SchemaUpdateOption.ALLOW_FIELD_ADDITION,
+    ]
+    # 'REQUIRED' fields cannot be added to an existing schema, so the
+    # additional column must be 'NULLABLE'
+    job_config.schema = [
+        bigquery.SchemaField('full_name', 'STRING', mode='REQUIRED'),
+        bigquery.SchemaField('age', 'INTEGER', mode='NULLABLE'),
+    ]
+    job_config.source_format = bigquery.SourceFormat.CSV
+    job_config.skip_leading_rows = 1
+
+    with open(filepath, 'rb') as source_file:
+        job = client.load_table_from_file(
+            source_file,
+            table_ref,
+            location='US',  # Must match the destination dataset location.
+            job_config=job_config)  # API request
+
+    job.result()  # Waits for table load to complete.
+
+    print('Loaded {} rows into {}:{}.'.format(
+        job.output_rows, dataset_id, table_ref.table_id))
+
+    # Verify that the column was added
+    table = client.get_table(table)
+    assert len(table.schema) == 2
+    # [END bigquery_add_column_load_append]
+    assert table.num_rows > 0
+
+
+def test_load_table_relax_column(client, to_delete):
+    dataset_id = 'load_table_relax_column_{}'.format(_millis())
+    dataset_ref = client.dataset(dataset_id)
+    dataset = bigquery.Dataset(dataset_ref)
+    dataset.location = 'US'
+    dataset = client.create_dataset(dataset)
+    to_delete.append(dataset)
+
+    snippets_dir = os.path.abspath(os.path.dirname(__file__))
+    filepath = os.path.join(
+        snippets_dir, '..', '..', 'bigquery', 'tests', 'data', 'people.csv')
+
+    # [START bigquery_relax_column_load_append]
+    # from google.cloud import bigquery
+    # client = bigquery.Client()
+    # dataset_ref = client.dataset('my_dataset')
+    # filepath = 'path/to/your_file.csv'
+
+    table_ref = dataset_ref.table('my_table')
+    old_schema = [
+        bigquery.SchemaField('full_name', 'STRING', mode='REQUIRED'),
+        bigquery.SchemaField('age', 'INTEGER', mode='REQUIRED'),
+        bigquery.SchemaField('favorite_color', 'STRING', mode='REQUIRED'),
+    ]
+    table = client.create_table(bigquery.Table(table_ref, schema=old_schema))
+
+    # Configure the load job to append the data to a destination table,
+    # allowing field relaxation
+    job_config = bigquery.LoadJobConfig()
+    job_config.write_disposition = bigquery.WriteDisposition.WRITE_APPEND
+    job_config.schema_update_options = [
+        bigquery.SchemaUpdateOption.ALLOW_FIELD_RELAXATION,
+    ]
+    # The source file data to load has one less column than the existing table
```

Comment:
Make this comment clearer that the missing column is required without the relaxation?